frontends/infinite-corridor/src/scss/main-search.scss

```diff
@@ -8,23 +8,35 @@ $max-width: 520px;
   &.MuiInputBase-root {
     max-width: $max-width;
     width: 100%;
-    border-radius: 100vh;
+    border-radius: 3px;
     font-size: 1.25rem;
 
-    &.Mui-focused fieldset {
-      border-color: theme.$color-gray-4;
+    fieldset {
+      border-color: theme.$color-blue-5;
+      border-width: 1.5px;
+    }
+
+    &.Mui-focused  fieldset{
+       border-color: theme.$color-blue-5;
     }
   }
+
+
   
   .MuiInputBase-input {
     &::placeholder {
       font-size: 1rem;
     }
   }
 
-  .search-icon-button {
-    border: solid 1px;
-    border-color: #A32034;
+  .MuiButton-root {
+    margin-right: -13px;
+    border-radius: 3px;
+    padding: 10px;
+    background-color: theme.$color-blue-5;
   }
 
+  .MuiButton-root:hover {
+    background-color: "#03152D"
+  }
```

Comment:
For color and MUI components, we're much better off passing the `color=` prop. You can't see my mouse in the screenshot, but I'm hovering the keyboard-focused search button:

<img width="625" alt="Screenshot 2023-03-17 at 1 17 21 PM" src="https://user-images.githubusercontent.com/9010790/225974099-ca30b259-3ccc-452e-b275-29e9af3b8003.png">

Above, MUI still thinks the components have `color="primary"` (the default), and that primary reddish color is still coming through sometimes.

My rec would be to remove all of the color stuff from `.main-search`, and set `color="secondary"` on the SearchInput's `OutlinedInput`, `Button`, and `IconButton`. _Or given `SearchInput` a color prop that gets forwarded to those three components._